src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -22,24 +22,28 @@
                       {:id :pending :label (i18n/label :t/pending) :accessibility-label :pending-tab}
                       {:id :opened :label (i18n/label :t/opened) :accessibility-label :opened-tab}]}]])
 
+(defn render-fn
+  [id]
+  (let [community (rf/sub [:communities/home-item id])]
+    [quo/communities-membership-list-item
+     {:on-press      #(rf/dispatch [:navigate-to-nav2 :community-overview id])
+      :on-long-press #(rf/dispatch
+                       [:bottom-sheet/show-sheet
+                        {:content       (fn []
+                                          [options/community-options-bottom-sheet id])
+                         :selected-item (fn []
+                                          [quo/communities-membership-list-item {} community])}])}
+     community]))
+
 (defn communities-list
   [communities-ids]
-  [rn/view
-   (map-indexed
-    (fn [index id]
-      (let [community (rf/sub [:communities/home-item id])]
-        ^{:key index}
-        [quo/communities-membership-list-item
-         {:on-press      #(rf/dispatch [:navigate-to-nav2 :community-overview id])
-          :on-long-press #(rf/dispatch
-                           [:bottom-sheet/show-sheet
-                            {:content       (fn []
-                                              [options/community-options-bottom-sheet id])
-                             :selected-item (fn []
-                                              [quo/communities-membership-list-item {} community])}])}
-         community]))
-    communities-ids)])
-
+  [rn/flat-list
+   {:key-fn                            :id
+    :keyboard-should-persist-taps      :always
+    :shows-horizontal-scroll-indicator false
+    :separator                         [rn/view {:width 12}]
+    :data                              communities-ids
+    :render-fn                         render-fn}])
```

Comment:
The `:render-fn` keyword often confuses us. The component we pass to `render-fn` is really a normal component, like any other, so I think we shouldn't name it `render-fn` or anything containing the word `render`. The only difference is that this component receives a particular set of arguments, like the current element under iteration and the index.

The component could be named just `community-list-item`.